lib/src/widgets/palette_picker.dart

```diff
@@ -93,13 +98,12 @@ class _PalettePickerState extends State<PalettePicker> {
     return ratio * maxposition + (1.0 - ratio) * minposition;
   }
 
+  static final defaultBorder = Border.all(color: Colors.grey);
```

Comment:
Actually now I'm not sure about my comment any more, since we lose the `const` before `PalettePicker`.
Maybe using `defaultBorderRadius` was a better idea.
@AlexV525 what do you think?

Reply:
Sorry for the late reply, it can be `Border.fromBorderSide`.